tests/base/src/test/java/org/keycloak/tests/admin/AdminRootTest.java

```diff
@@ -63,7 +63,9 @@ public void testRedirect(String hostname) throws Exception {
 
     @Test
     public void testNoRedirectWithFrontendUrl() throws Exception {
-        HttpResponse response = client.execute(new HttpGet(HOSTNAME + "/admin"));
+        HttpGet request = new HttpGet(HOSTNAME + "/admin");
+        request.addHeader("Forwarded", "for=192.0.2.60");
```

Comment:
It'd be good to have some comment here that we're adding an arbitrary Forwarded header to make the `WelcomeResource.isLocal` check fail.